doc/api/cli.md

```diff
@@ -315,6 +307,14 @@ added: v11.8.0
 
 Use the specified file as a security policy.
 
+### `--no-experimental-fetch`
```

Comment:
Why do we need an opt-out flag?

Reply:
I was thinking it could be useful for people who depend on polyfills that install another version of `fetch` with a different feature set.

Reply:
XHR